config/scss-lint.yml

```diff
@@ -0,0 +1,208 @@
+# Default application configuration that all configurations inherit from.
```

Comment:
@rustemginiyatullin no need to copy default configuration b/c it will be loaded anyway. In `config/scss-lint.yml` we should define only overwrites of the default configuration.